go/cmd/vtctldclient/command/tablets.go

```diff
@@ -190,6 +192,15 @@ Note that, in the SleepTablet implementation, the value should be positively-sig
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandStopReplication,
 	}
+	// GetPermissions asks the remote tablet for its permissions list.
+	GetPermissions = &cobra.Command{
+		Use:                   "GetPermissions <tablet_alias>",
+		Aliases:               []string{"GetPermissions"},
+		Short:                 "Asks the remote tablet for its permissions list.",
```

Comment:
A better help message to the user would be something more like: "Returns the permissions list from the remote tablet." (or whatever is in `go/vt/vtctl/` for this equivalent)